next/components/forms/icon-components/Waves.tsx

```diff
@@ -0,0 +1,214 @@
+type WaveProps = {
+  className?: string
+  waveColor?: string
+  innerLinesColor?: string
+  outerLinesColor?: string
+  isRich?: boolean
+}
+
+const Waves = ({
```

Comment:
Please get the newer version of Wawes from bratislava.sk project. This file is too old an doutdated.